docs/pages/setup/reference/authentication.mdx

```diff
@@ -1,22 +1,21 @@
 ---
 title: Authentication options
-description: Teleport enterprise license file configuration parameters and requirements
+description: A reference for Teleport's authentication connectors
 ---
 
-Teleport uses the concept of "authentication connectors" to authenticate users
-when they execute [`tsh login`](./cli.mdx#tsh-login) command. There are three
-types of authentication connectors:
+Teleport uses the concept of **authentication connectors** to authenticate users
+when they execute the [`tsh login`](./cli.mdx#tsh-login) command. There are three
+types of authentication connectors.
 
 ## Local
```

Comment:
If we want to be accurate, I don't think local users use an auth connector. An auth connector is for when you want to authenticate against an _external_ identity provider. If you want to authenticated against teleport's internal user DB, then you don't need to create an auth connector at all.

Reply:
It looks like a couple other parts of the Teleport docs/product mention a "local" auth connector. 

- [Enterprise SSO guide](https://github.com/gravitational/teleport/blame/75e24cbd54ccb7850d257be1dad59208a4179d38/docs/pages/enterprise/sso.mdx#L61)
- The help text for the `auth` flag of `tsh` (https://goteleport.com/docs/setup/reference/cli/#tsh) seems to imply that `local` is a kind of auth connector

Should we be more careful here and in those other locations to distinguish `local` form actual auth connectors? 

For this PR, I've suggested changing this H2 to "Local (no authentication connector)".

Reply:
Yes, I think we should be more careful throughout the docs.

If you go to the "auth connectors" tab of the web UI, you won't see one for local, so referring to it as the "Local Auth Connector" is likely to confuse people.

Reply:
👍 Added an issue: https://github.com/gravitational/teleport/issues/10761